doc/code/qml_variable.rst

```diff
@@ -6,7 +6,7 @@ qml.variable
 .. warning::
 
     Unless you are a PennyLane or plugin developer, you likely do not need
-    to use the ``VariableRef`` class.
+    to use the ``Variable`` class.
 
 .. automodapi:: pennylane.variable

```

Comment:
Just curious, wouuld this line actually not be auto-generating the expected content after the `VariableRef` rename? (i.e., should have been `pennylane.variableref`?)

Reply:
Ahh nevermind, the file was still called `variable.py`, so shouldn't have been an issue